keywords.txt

```diff
@@ -28,4 +28,7 @@ led 			KEYWORD2
 
 SWAP_LEFT		LITERAL1
 SWAP_RIGHT		LITERAL1
-SWAP_BOTH		LITERAL1
\ No newline at end of file
+SWAP_BOTH		LITERAL1
+MMAX_3_OHM     LITERAL1
+MMAX_16_OHM        LITERAL1
+PWM_OF_START_MOVING     LITERAL1
```

Comment:
Tab/spaces inconsistency. Use either, not both

Reply:
Yes, auto-replacer worked. Switch-off.